common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -304,16 +304,20 @@ def get_html(self):
             if xblock_settings and 'YOUTUBE_API_KEY' in xblock_settings:
                 yt_api_key = xblock_settings['YOUTUBE_API_KEY']
 
+        poster = None
+        if edxval_api and self.edx_video_id:
+            poster = edxval_api.get_course_video_image_url(
+                course_id=self.runtime.course_id.for_branch(None),
+                edx_video_id=self.edx_video_id.strip()
```

Comment:
@muhammad-ammar I think we need to change this as well, if you look at line https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/video_module/video_module.py#L517
it's check for id before making a call.